bin/aws-infrastructure.ts

```diff
@@ -1,26 +1,22 @@
 #!/usr/bin/env node
 import * as config from 'config';
-import 'source-map-support/register';
 import * as cdk from 'aws-cdk-lib';
 import { SaasInfrastructureStack, VpcStackProps } from '../lib/saas-infrastructure-stack';
 import { VpcInfrastructureStack } from '../lib/vpc-infrastructure-stack';
-import { NelsonLoginProviderStack } from '../lib/nelson-login-provider-stack';
 
 const app = new cdk.App();
 
 var vpcprops: VpcStackProps = {};
-if(config.get('useexistingvpc') == true && config.has('existingvpcid')) {
-  vpcprops.vpcid = config.get('existingvpcid') as String;
+if (config.get('useexistingvpc') == true && config.has('existingvpcname')) {
+    vpcprops.vpcname = config.get('existingvpcname') as string;
 }
 else {
-  const vpcStack = new VpcInfrastructureStack(app, `${config.get('environmentname')}Vpc`, {});
-  // vpcprops.cfnvpc = vpcStack.getCfnVpc();
```

Comment:
Please remove commented code. We should only push comments when it is instructions.